medical_appointment/models/medical_appointment.py

```diff
@@ -55,9 +56,10 @@ class MedicalAppointment(models.Model):
         help='When to stop displaying appointment',
     )
     duration = fields.Float(
-        string='Duration',
+        string='Duration (min)',
         help='Duration of appointment (in minutes)',
         default=30.0,
```

Comment:
Add a `digits`, with a decimal precision specific for appointment times.

https://github.com/odoo/odoo/blob/fc2e80cb4bcc450762c7ac5cb82a3e2d88062b38/addons/l10n_fr_hr_payroll/models/l10n_fr_hr_payroll.py#L11

https://github.com/odoo/odoo/blob/fc2e80cb4bcc450762c7ac5cb82a3e2d88062b38/addons/hr_payroll/data/hr_payroll_data.xml#L82